CONTRIBUTING.md

```diff
@@ -3,12 +3,12 @@
 Contributions to this Module are very welcome! We follow a fairly standard [pull request 
 process](https://help.github.com/articles/about-pull-requests/) for contributions, subject to the following guidelines:
  
-1. [File a GitHub issue](#file-a-github-issue)
-1. [Update the documentation](#update-the-documentation)
-1. [Update the tests](#update-the-tests)
-1. [Update the code](#update-the-code)
-1. [Create a pull request](#create-a-pull-request)
-1. [Merge and release](#merge-and-release)
+ 1. [File a GitHub issue](#file-a-github-issue)
+ 2. [Update the documentation](#update-the-documentation)
+ 3. [Update the tests](#update-the-tests)
+ 4. [Update the code](#update-the-code)
+ 5. [Create a pull request](#create-a-pull-request)
+ 6. [Merge and release](#merge-and-release)
```

Comment:
I don't think you need to change the numbers. Markdown rendering will automatically render these properly:

![image](https://user-images.githubusercontent.com/34349331/72459535-fbf64980-3798-11ea-8785-f2c55fa4ed12.png)

Keeping them as `1.` makes it easier to rearrange things or add something in the middle without having to re-number everything yourself.